addons/packages/whereabouts/0.5.0/repo/.imgpkg/images.yml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
What is the usage of this `repo` folder?

Reply:
This is a file for testing, will remove it!